### PR TITLE
[WIP] Add /ping

### PR DESCRIPTION
**Pull request by giorgiosironi**
Goal: add a /ping API that just returns the text `pong` plus a few "don't cache anything" headers.

### CODE REVIEW

File: src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * @RestResource(
+ *   id = "ping_rest_resource",
+ *   label = @Translation("Ping rest resource"),
+ *   uri_paths = {
+ *     "canonical" = "/ping"
+ *   }
+ * )
+ */
+class PingRestResource extends AbstractRestResourceBase {
```
Comment (thewilkybarkid) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: There must be a simpler way to add a controller than extending whatever `AbstractRestResourceBase` is?

Comment (nlisgo) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: There is a simpler way. But to complicate things slightly I think we should have this in it's own module.

The missing step to get this working is that we need to set the permissions to access the resource. This is typically done in the UI and then we can export the configuration.

@giorgiosironi let's discuss which is the best approach to implement this. Do we want it to always be available or is something we could enable, test, then tear down?

Comment (giorgiosironi) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: Probably, @nlisgo shall I use the @RestResource support or can I go for something simpler like https://www.drupal.org/docs/8/creating-custom-modules/adding-a-basic-controller ? In which module should this be added? Does it need a new one?

Comment (nlisgo) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: I think it needs a new one. Would `jmcs_ping` be an appropriate name?

Comment (thewilkybarkid) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: We'd probably want it on any Drupal, so maybe just `ping`? (Or `elife_ping` due to https://www.drupal.org/project/ping?)

Comment (giorgiosironi) [inline src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php]: always available, we use it in our /status page

**Comment by giorgiosironi** [comment]
Obsolete by #137.

**Pull request closed without merging**